### PR TITLE
linux-mainline: Upgrade to the 5.4 kernel

### PR DESCRIPTION
**Pull request by alistair23**

### CODE REVIEW

File: recipes-kernel/linux/linux-mainline_5.4.bb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 require recipes-kernel/linux/linux-mainline-common.inc
 
-LINUX_VERSION ?= "5.3.x"
+LINUX_VERSION ?= "5.4.x"
 KERNEL_VERSION_SANITY_SKIP="1"
 
-BRANCH = "linux-5.3.y"
+# BRANCH = "linux-5.4.y"
+BRANCH = "master"
```
Comment (kraj) [inline recipes-kernel/linux/linux-mainline_5.4.bb]: hmmm master seems wrong here

Comment (alistair23) [reply recipes-kernel/linux/linux-mainline_5.4.bb]: Yep, just waiting for the stable branch to be released once 5.4 comes out. Then I'll update this and change the PR from draft state. I just wanted to put this out there for anyone who wants to test.

**Comment by kraj** [comment]
marking with `WIP` `DO-NOT-MERGE`or `RFT` would help to mark it as such

**Comment by alistair23** [comment]
Sorry, I was using the new draft PRs, so you shouldn't be able to merge it. Then when it's ready I can just mark it as ready and not have to edit the PR name. I'll add WIP to it now.

**Comment by alistair23** [comment]
This is ready to merge.

**Pull request merged**